<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PokedexWindow</class>

 <widget class="QMainWindow" name="PokedexWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>650</height>
   </rect>
  </property>

  <property name="windowTitle">
   <string>Offline Pokedex</string>
  </property>

  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="mainLayout">

    <!-- ================= LEFT PANEL ================= -->
    <item>
     <widget class="QWidget" name="leftPanel">
      <property name="minimumWidth">
       <number>260</number>
      </property>
      <property name="maximumWidth">
       <number>260</number>
      </property>

      <layout class="QVBoxLayout" name="leftLayout">
       <property name="spacing">
        <number>7</number>
       </property>

       <!-- LOGO (PURE UI, NO CODE) -->
       <item>
        <widget class="QLabel" name="lblLogo">
         <property name="minimumHeight">
          <number>80</number>
         </property>
         <property name="pixmap">
          <pixmap>images/pokedex_logo.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>

       <!-- SEARCH -->
       <item>
        <widget class="QLineEdit" name="searchEdit">
         <property name="styleSheet">
          <string notr="true">
QLineEdit {
 font-size: 15px;
 padding: 8px 12px;
 border-radius: 12px;
 border: 1px solid #999;
 background: #ffffff;
 color: #000000;
}
QLineEdit::placeholder {
 color: #666666;
 font-style: italic;
}
QLineEdit:focus {
 border: 2px solid #4CA3E0;
}
          </string>
         </property>
         <property name="placeholderText">
          <string>Search Pokémon...</string>
         </property>
        </widget>
       </item>

       <!-- LIST -->
       <item>
        <widget class="QListWidget" name="pokemonList"/>
       </item>

      </layout>
     </widget>
    </item>

    <!-- ================= RIGHT PANEL ================= -->
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>

      <widget class="QWidget" name="scrollWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>800</width>
         <height>620</height>
        </rect>
       </property>

       <layout class="QVBoxLayout" name="rightLayout">

        <!-- TITLE -->
        <item>
         <widget class="QLabel" name="lblName">
          <property name="styleSheet">
           <string>font-size:22px;font-weight:bold;</string>
          </property>
          <property name="text">
           <string>Select Pokémon</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>

        <!-- IMAGE -->
        <item>
         <widget class="QLabel" name="lblImage">
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>

        <!-- INFO CARD -->
        <item>
         <widget class="QFrame" name="infoCard">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>

          <layout class="QVBoxLayout" name="infoLayout">

           <item>
            <layout class="QHBoxLayout">
             <item><widget class="QLabel" name="lblHeight"/></item>
             <item><widget class="QLabel" name="lblCategory"/></item>
            </layout>
           </item>

           <item>
            <layout class="QHBoxLayout">
             <item><widget class="QLabel" name="lblWeight"/></item>
             <item><widget class="QLabel" name="lblAbilities"/></item>
            </layout>
           </item>

           <item>
            <layout class="QHBoxLayout">
             <item><widget class="QLabel" name="lblGender"/></item>
             <item>
              <spacer name="spacer1">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>

          </layout>
         </widget>
        </item>

        <!-- STATS -->
        <item>
         <widget class="QLabel" name="lblStatsTitle">
          <property name="styleSheet">
           <string>font-weight:bold;</string>
          </property>
          <property name="text">
           <string>Stats</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>

        <item><widget class="QProgressBar" name="barHP"/></item>
        <item><widget class="QProgressBar" name="barATK"/></item>
        <item><widget class="QProgressBar" name="barDEF"/></item>
        <item><widget class="QProgressBar" name="barSPA"/></item>
        <item><widget class="QProgressBar" name="barSPD"/></item>
        <item><widget class="QProgressBar" name="barSPE"/></item>

        <!-- TYPE -->
        <item>
         <widget class="QLabel" name="lblTypeTitle">
          <property name="styleSheet">
           <string>font-weight:bold;</string>
          </property>
          <property name="text">
           <string>Type</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>

        <item>
         <layout class="QHBoxLayout" name="typeLayout"/>
        </item>

        <!-- WEAKNESS -->
        <item>
         <widget class="QLabel" name="lblWeakTitle">
          <property name="styleSheet">
           <string>font-weight:bold;</string>
          </property>
          <property name="text">
           <string>Weaknesses</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>

        <item>
         <layout class="QHBoxLayout" name="weakLayout"/>
        </item>

        <!-- EVOLUTION -->
        <item>
         <widget class="QLabel" name="lblEvoTitle">
          <property name="styleSheet">
           <string>font-weight:bold;</string>
          </property>
          <property name="text">
           <string>Evolutions</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>

        <item>
         <layout class="QHBoxLayout" name="evoLayout"/>
        </item>

        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>

       </layout>
      </widget>
     </widget>
    </item>

   </layout>
  </widget>
 </widget>

 <resources/>
 <connections/>
</ui>